molecules/_tables.scss

```diff
@@ -51,37 +51,41 @@ table {
     border-bottom: 1px solid $table-header-border-color;
     background-color: $table-header-background-color;
     color: $table-header-color;
-    th {
-      text-align: left;
-    }
-    tr:not(:last-child) th {
-      border-bottom: 1px solid $table-header-row-border-bottom-color;
-    }
-    tr:not(:first-child) th {
-      border-top: 1px solid $table-header-row-border-top-color;
-      background-color: darken($table-header-background-color, 3%);
-    }
-    tr:last-child th {
-      border-bottom: 1px solid $table-header-border-color;
-    }
-    tr:first-child {
+    tr {
       th {
         padding: 0 $space-xl 0 $space-m;
         line-height: $space-xl;
         white-space: nowrap;
         position: relative;
+        text-align: left;
+        border-bottom: 1px solid $table-header-border-color;
         label {
           font-size: rem(13px);
           font-weight: normal;
           text-transform: uppercase;
           display: inline-block;
+          vertical-align: top;
           margin: 0;
+          max-width: 100%;
+          overflow: hidden;
+          text-overflow: ellipsis;
         }
         .sort {
```

Comment:
@fragsalat which element should take those classes `.sort` and `.filter` ? I'm a bit confused because in the top `.sort` has position:relative and width:10px I guess that's a container for the arrow icon...right ?